weareinreach/inreach-catalog
# Feature/1202137657447314/update mx crime data

## Description
updated the numbers used when displaying the Mexico Crime Data
added a link to the data legend - goes to the source data site 

## Asana ticket:
https://app.asana.com/0/1132189118126148/1202137657447314

## PR Checklist




## How to Test

1. select location as Mexico
2. toggle the 'Generalized Murder' button on the map
3. ensure crime data is highlighted

<img width="372" alt="Screen Shot 2022-04-19 at 9 09 29 AM" src="https://user-images.githubusercontent.com/10123216/164047941-b96f2ef7-a353-45dd-b131-1056ed71546d.png">
